content/EngineeringPractice-1.md

```diff
@@ -0,0 +1,62 @@
+Title: Engineering Practice Nuggets
+Date: 2023-01-10
+Category: Posts 
+Tags: agile, architecture, code, code-quality, design, engineering, technical-excellence, tdd, tips, learning, innovation
+Slug: EngineeringPractice-1
+Author: Martin M. Lacey
+Summary: Nuggets of Knowledge and Best Practices in Software Engineering
+
+When we design, build, and deploy software we aim to apply patterns and learned best practices in order to produce
+consistent and rohbust solutions that are of the highest quality and lowest possible technical dept (burden to maintain).
+
+These powerful nuggets of knowledge, patterns and practices, and key areas that we are evolving and 
+improving will be the subject matter explored with the goal to invigorate 
+vibrant collaboration and innovation, and indeed experimentation and tenacity 
+when things don't go as desired.
+
+
+---
+# Metrics and Quantitative Measures
+We have begun the process of calculating various metrics on software created at
```

Comment:
You could mention the four DevOps metrics, which are standard across DevOps ecosystem.